src/stavetempo.ts

```diff
@@ -100,7 +100,7 @@ export class StaveTempo extends StaveModifier {
 
       x += 3 * scale;
       Glyph.renderGlyph(ctx, x, y, options.glyph_font_scale, glyphProps.code_head);
-      x += glyphProps.getWidth ? glyphProps.getWidth() : 0 * scale;
+      x += Glyph.getWidth(glyphProps.code_head, options.glyph_font_scale);
```

Comment:
Was the previous approach buggy? Or is this just a cleaner way to do things?

Reply:
Yes, the notehead and stem of the stavetempo were not connected. This is what I meant with the second bulltet:

> Looking in detail at all the tests, I found two bugs:
> 
> * clef code used for percussion was wrong, now fixed.
> * stavetempo had not been aligned with [custom noteheads refactored #1477](https://github.com/0xfe/vexflow/pull/1477), now fixed

Reference
![Stave Tempo_Test Bravura](https://user-images.githubusercontent.com/22865285/205947978-f54d837e-2279-4395-855d-53f6caa32584.png)
Current
![Stave Tempo_Test Bravura](https://user-images.githubusercontent.com/22865285/205948168-918cfff5-ed3e-4678-b3e6-dd8b182c06f6.png)